<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThreeChoicesWindow</class>
 <widget class="QWidget" name="ThreeChoicesWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ThreeChoicesWindow</string>
  </property>
  <widget class="QWidget" name="widget_background" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>560</height>
    </rect>
   </property>
   <widget class="QPushButton" name="btn_continue">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>200</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_create">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>200</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_display">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>200</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>btn_create</zorder>
   <zorder>btn_display</zorder>
   <zorder>btn_continue</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
